docs/source/using/troubleshooting.adoc

```diff
@@ -91,23 +109,21 @@ system.
 NOTE: You can completely reset the IP database by removing the files
 manually but this is very *risky*.
 
-[[troubleshooting-vbox]]
-== VirtualBox driver
-
 [[machine-doesnt-exist]]
-=== Error: getting state for host: machine does not exist
+=== VirtualBox: `Error: getting state for host: machine does not exist`
 
 If you use Windows, make sure that you set the `--vm-driver virtualbox`
 flag in the `minishift start` command. Alternatively, the problem might be
 an outdated version of VirtualBox.
 
 To avoid this issue, it is recommended to use VirtualBox 5.1.12 or later.
 
-[[troubleshooting-authentication]]
-== Users and authentication
+
+[[troubleshooting-general-problems]]
+== Troubleshooting general problems
 
 [[special-characters-passwords]]
-=== Some special characters cause passwords to fail
+=== Users and authentication: Some special characters cause passwords to fail
```

Comment:
I think 'Users and authentication:' is not required. "special characters cause passwords to fail" should do.